### PR TITLE
Added enumeration instead of private static fields.

### PR DESCRIPTION
**Pull request by SwiftySwizzy**
It is better to use enumeration instead of static private fields. It is described in Effective Java Book, item 30, page 147.

### CODE REVIEW

**Comment by chesterbr** [comment]
Hello @SwiftySwizzy. Indeed they are, but this change also changes comments and other places where it shouldn't. The name is also a bit odd (Qualificacao), could you clarify why it was chosen?

I'm also wondering whether this relates in any sense to #12.

**Comment by SwiftySwizzy** [comment]
Yes it is related. We are doing it with a friend of mine. Will be fixed the usage of enums as in #12.

**Comment by SwiftySwizzy** [comment]
I have removed more of the if statements that were pointless in one of the classes. Further improvements suggest fixing the JavaDoc in the class mentioned above and enumeration fix.

File: app/src/main/java/me/chester/minitruco/core/EstrategiaGasparotto.java [hunk 1]
```diff
@@ -26,9 +26,9 @@
  *   software sem autorização prévia específica por escrito.
  * 
  * ESTE SOFTWARE É FORNECIDO PELOS DETENTORES DE DIREITOS AUTORAIS E
- * CONTRIBUIDORES "COMO ESTÁ", ISENTO DE GARANTIAS EXPRESSAS OU TÁCITAS,
- * INCLUINDO, SEM LIMITAÇÃO, QUAISQUER GARANTIAS IMPLÍCITAS DE
- * COMERCIABILIDADE OU DE ADEQUAÇÃO A FINALIDADES ESPECÍFICAS. EM NENHUMA
+ * CONTRIBUIDORES "COMO ESTÁ", ISENTO DE GARANTIAS.value() EXPRESSAS.value() OU TÁCITAS.value(),
+ * INCLUINDO, SEM LIMITAÇÃO, QUAISQUER GARANTIAS.value() IMPLÍCITAS.value() DE
+ * COMERCIABILIDADE OU DE ADEQUAÇÃO A FINALIDADES ESPECÍFICAS.value(). EM NENHUMA
```
Comment (chesterbr) [inline app/src/main/java/me/chester/minitruco/core/EstrategiaGasparotto.java]: ^ these `value()` are probably a find/replace gone wrong.

File: app/src/main/java/me/chester/minitruco/core/EstrategiaGasparotto.java [hunk 2]
```diff
@@ -483,59 +490,59 @@ private boolean partidaGanhaParaAdversario(SituacaoJogo s) {
 							.getValorTruco(s.manilha)) {
 					case 11:
 						PICAFUMOJaSaiu = true;
-						break; // PICAFUMO já saiu
+						break; // PICAFUMO.value() já saiu
```
Comment (chesterbr) [inline app/src/main/java/me/chester/minitruco/core/EstrategiaGasparotto.java]: This comment change (and equivalents below) isn't necessary.

File: app/src/main/java/me/chester/minitruco/core/EstrategiaGasparotto.java [hunk 3]
```diff
@@ -622,14 +629,14 @@ else if (carta.getValorTruco(s.manilha) == 14)
...
+			qcarta = Qualificacao.TRES.value();
+		if (qcarta == Qualificacao.AS.value() && tres_testedemanilha.getValorTruco(s.manilha) == 14)
+			qcarta = Qualificacao.DOIS.value();
 		// caso a manilha seja um 2, o A passa a ter valor de 2!
-		if (qcarta == AS && dois_testedemanilha.getValorTruco(s.manilha) == 14)
-			qcarta = DOIS;
+		if (qcarta == Qualificacao.AS.value() && dois_testedemanilha.getValorTruco(s.manilha) == 14)
+			qcarta = Qualificacao.DOIS.value();
 
 		return qcarta;
```
Comment (chesterbr) [inline app/src/main/java/me/chester/minitruco/core/EstrategiaGasparotto.java]: What do you think about declaring `qcarta` as a `Qualificacao` and just converting to integer in the end?

Alternative: changing the return type of this method to `Qualificacao` (I'm not sure *all* uses of it are compatible with that, but the ones below look like so)

File: app/src/main/java/me/chester/minitruco/core/EstrategiaGasparotto.java [hunk 4]
```diff
@@ -1063,7 +1070,7 @@ private int joga_primeira_rodada_parape(SituacaoJogo s) {
 		// caso tenhamos uma manilha (menos o zap) para tentar fazer
 		// e não dar chance ao pé de amarrar...
 		// assim como outros casos... mas tudo bem...
-		// Se for lixo (menor que ás), vamos reforçar.
+		// Se for LIXO.value() (menor que ás), vamos reforçar.
```
Comment (chesterbr) [inline app/src/main/java/me/chester/minitruco/core/EstrategiaGasparotto.java]: More find/replace leftovers here and below.

**Pull request closed without merging**